ingestion_server/ingestion_server/ingest.py

```diff
@@ -229,10 +231,7 @@ def reload_upstream(table, progress=None, finish_time=None):
         DROP SCHEMA IF EXISTS upstream_schema CASCADE;
         CREATE SCHEMA upstream_schema AUTHORIZATION deploy;
 
-        IMPORT FOREIGN SCHEMA public
-          LIMIT TO (
-            {table}, image_normalized_popularity
-          ) FROM SERVER upstream INTO upstream_schema;
+        IMPORT FOREIGN SCHEMA public FROM SERVER upstream INTO upstream_schema;
```

Comment:
You could probably bring back a `LIMIT TO` statement here, since now the view is materialized, and no access to any other functions or data is necessary.